tensorflow/compiler/xla/tests/fusion_test.cc

```diff
@@ -522,6 +522,39 @@ XLA_TEST_F(FusionTest, DISABLED_ON_CPU(ReduceWindow)) {
       *ExecuteAndTransfer(std::move(hlo_module), {}));
 }
 
+XLA_TEST_F(FusionTest, SharedConstant) {
+  auto hlo_module = CreateNewModule();
```

Comment:
Can you add a comment here explaining what this test is testing, on a high-level?